client/app/index.html

```diff
@@ -503,7 +503,7 @@ <h2 translate>
               </span>
               <span class="md-subhead description">
                   <translate class="balance">Balance</translate> {{ul.network.symbol}}{{ul.selected.balance/100000000}}
-                  <md-button ng-click="ul.selectNextCurrency()">{{ul.formatCurrencyBalance(ul.selected.balance/100000000)}}</md-button>
+                  <md-button>{{ul.formatCurrencyBalance(ul.selected.balance/100000000)}}</md-button>
```

Comment:
Is there another element we can change this to, since it's no longer a usable button?

Reply:
Ye, was thinking of just removing the button and styling it a bit better.